apps/syntrest/src/main/java/no/nav/registre/syntrest/config/AppConfig.java

```diff
@@ -4,81 +4,50 @@
 import io.kubernetes.client.apis.CustomObjectsApi;
 import io.kubernetes.client.util.Config;
 import io.kubernetes.client.util.KubeConfig;
+import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
-import no.nav.registre.syntrest.consumer.SyntAmeldingConsumer;
-import no.nav.registre.syntrest.consumer.SyntConsumer;
-import no.nav.registre.syntrest.kubernetes.ApplicationManager;
-import no.nav.registre.syntrest.kubernetes.KubernetesController;
 import no.nav.registre.testnorge.libs.core.config.ApplicationCoreConfig;
 
-import org.apache.http.client.config.RequestConfig;
-import org.apache.http.conn.ssl.DefaultHostnameVerifier;
-import org.apache.http.impl.client.HttpClientBuilder;
-import org.apache.http.impl.conn.SystemDefaultRoutePlanner;
 import org.springframework.beans.factory.BeanCreationException;
 import org.springframework.beans.factory.annotation.Value;
-import org.springframework.boot.web.client.RestTemplateBuilder;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.DependsOn;
 import org.springframework.context.annotation.Import;
-import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
-import org.springframework.web.client.RestTemplate;
+import org.springframework.context.annotation.Profile;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.springframework.web.util.DefaultUriBuilderFactory;
+import org.springframework.web.util.UriBuilderFactory;
 
 import java.io.FileReader;
 import java.io.IOException;
-import java.net.ProxySelector;
 import java.util.concurrent.Executors;
 import java.util.concurrent.ScheduledExecutorService;
 
 @Slf4j
 @Configuration
+@RequiredArgsConstructor
 @Import(ApplicationCoreConfig.class)
+@Profile("!test")
 public class AppConfig {
 
-    private static final int TIMEOUT = 240_000;
-    private final int EXECUTOR_POOL_SIZE = 4;
     @Value("${kube-config-path}")
     private String kubeConfigPath;
-    @Value("${isAlive}")
-    private String isAliveUrl;
-    @Value("${docker-image-path}")
-    private String dockerImagePath;
-    @Value("${max-alive-retries}")
-    private int maxRetries;
-    @Value("${alive-retry-delay}")
-    private int retryDelay;
-    @Value("${github_username}")
-    private String github_username;
-    @Value("${github_password}")
-    private String github_password;
-    @Value("${proxy-url}")
-    private String proxyUrl;
-    @Value("${proxy-port}")
-    private int proxyPort;
-    @Value("${synth-amelding-url}")
-    private String ameldingUrl;
 
     @Bean
     ScheduledExecutorService scheduledExecutorService() {
-        return Executors.newScheduledThreadPool(EXECUTOR_POOL_SIZE);
+        int executorPoolSize = 4;
+        return Executors.newScheduledThreadPool(executorPoolSize);
```

Comment:
Tror ikke det er nødvendig å ha en egen variabel for `excutorPoolSize` ;)

Reply:
Den er bare en beskrivende variabel for å unngå magiske/uleselige parametere.
var tidligere definert på utsiden men sonarlint vil ha den inn, siden den bare brukes i denne beanen